### PR TITLE
json-rpc web3_sha3

### PR DESCRIPTION
**Pull request by NikVolf**

### CODE REVIEW

File: rpc/src/v1/impls/web3.rs [hunk 1]
```diff
@@ -34,4 +36,15 @@ impl Web3 for Web3Client {
 			_ => Err(Error::invalid_params())
 		}
 	}
+
+	fn sha3(&self, params: Params) -> Result<Value, Error> {
+		from_params::<(Bytes,)>(params).and_then(
+			|(data,)| {
+				let sha3 = data.to_vec().sha3();
```
Comment (arkpar) [inline rpc/src/v1/impls/web3.rs]: What type is `data` here? Is there a copy created in `to_vec`?

Comment (NikVolf) [inline rpc/src/v1/impls/web3.rs]: it's a byte array as i guess all the `data` specified in the json-rpc
there is only one example in wiki and it's a hex-encoded bytes

**Comment by NikVolf** [comment]
@arkpar added wiki test, passes

**Comment by gavofyork** [comment]
only removed two new lines so not bothering to redo CI.

**Pull request merged**